numpy/core/fromnumeric.py

```diff
@@ -62,6 +60,26 @@ def _wrapfunc(obj, method, *args, **kwds):
         return _wrapit(obj, method, *args, **kwds)
 
 
+def _wrapreduction(obj, method, name, axis, dtype, out, **kwargs):
+    passkwargs = {}
+    for k, v in kwargs.items():
+        if v is not np._NoValue:
+            passkwargs[k] = v
+
+    if type(obj) is not mu.ndarray:
+        try:
+            reduction = getattr(obj, name)
+        except AttributeError:
+            pass
+        else:
+            if dtype is not None:
```

Comment:
This could do with a comment justifying it - especially since it wasn't needed previously